docs/user_guide/HOWTO/Manage_TPM.rst

```diff
@@ -398,31 +400,30 @@ following:
 #. Update GRUB
 #. Reboot into a measured state
 
-For more information about tboot in general, reference external documentation:
+For more information about ``tboot`` in general, see:
 
-* https://fedoraproject.org/wiki/Tboot
 * The ``tboot`` docs found in ``/usr/share/tboot-*/*``
+* https://fedoraproject.org/wiki/Tboot
 * https://wiki.gentoo.org/wiki/Trusted_Boot
 * https://software.intel.com/sites/default/files/managed/2f/7f/Config_Guide_for_Trusted_Compute_Pools_in_RHEL_OpenStack_Platform.pdf
 
 
 Steps
 ^^^^^
 
-#. Enable Intel TXT and VT-d in the BIOS.
+#. Enable Intel ``TXT`` and ``VT-d`` in the BIOS.
 
 #. Boot into the kernel you want to trust (do not worry, this kernel will be
    preserved!)
 
-#. Follow the instructions in 'Starting With TPM' and ensure:
+#. Follow the instructions in :ref:`Starting with TPM` and ensure:
```

Comment:
this reference issues a warning:
`WARNING: undefined label: starting with tpm (if the link has no caption the label must precede a section header)`